### DISCUSSION TITLE
[V5] Is it possible to choose default serializing of ORM Model.

### DISCUSSION THREAD

**Discussion by hmziqrs** [adonisjs/core#2064] [Help] [answered]
https://preview.adonisjs.com/guides/models/serializing-models#points-to-note

According to the docs by default serializing converts model with snake_case.
In my project I don't consume snake_case data & I've to manually write additional `serializeAs` line with every property of model to normalize my response.
Is there any better way to configure to the default behavior ?

It's my first time trying adonis so sorry if it's already mentioned somewhere I missed it.

**Comment by thetutlage** [comment, marked as answer]
Inside `config/database.ts` file, under the `orm` object you can define a custom method.

```ts
orm: {
    getSerializeAsKey(_, key) {
      return key
    },
  },
```

**Reply by hmziqrs** [reply]
Thank you.
